test/robot/test_static_viewer.txt

```diff
@@ -0,0 +1,35 @@
+
+*** Settings ***
+
+Library           Selenium2Library
+
+
+
+*** Test Cases ***
+
+Test Static Viewer
+    Open Browser                        http://localhost:4080/webtest/static_example/       Chrome
```

Comment:
@will-moore thanks for writing the test, would it be possible to replace the above with the standard setup like https://github.com/openmicroscopy/openmicroscopy/blob/develop/components/tests/ui/testcases/web/view_image.txt#L10